downstream/modules/hub/proc-delete-collection.adoc

```diff
@@ -7,16 +7,19 @@ You can further manage your collections by deleting unwanted  collections, provi
 
 .Prerequisites
 * The collection being deleted does not have dependencies with other collections.
-* You have *Delete Collections* permissions
+* You have *Delete Collections* permissions.
 
 .Procedure
 . Log in to {PlatformName}.
 . Navigate to menu:Automation Hub[Collections].
```

Comment:
I found this in the IBM Style guide for "Navigate":

"Use only when a more direct verb, such as click, select, browse, or go to, is not applicable. Use only to provide direction to a location (for example, “Navigate to the directory that contains the files that you want to edit”)."

Not sure if in this context "go to" is better than "navigate" but just highlighting it :)

Reply:
Hi Michelle! Thanks for catching this one. Going to change it based on our team discussion.